Added a new note under Options for .NET Core 2.x

amaldevv opened the pull request:
## Summary

Web projects are not packable by default from .NET core 2.x. So, I added this under the options section as a note and also provided the code snippet to override this default behavior

amaldevv committed:
Added a new note under Options for .NET Core 2.x

Web projects are not packable by default from .NET core 2.x. So, I added this under the options section as a note and also provided the code snippet to override this default behavior

mairaw commented on docs/core/tools/dotnet-pack.md:
```diff
@@ -102,6 +102,13 @@ Defines the value for the `$(VersionSuffix)` MSBuild property in the project.
 
 Sets the verbosity level of the command. Allowed values are `q[uiet]`, `m[inimal]`, `n[ormal]`, `d[etailed]`, and `diag[nostic]`.
+> [!NOTE]
+> Web projects are not packable by default. To ovverride the default behaviour, add the below property to your `csproj` file
```
typo: ovverride

scottaddie replied:
- are not --> aren't
- `csproj` --> *.csproj*
- @KathleenDollard I wonder if the 1st sentence should say "Web projects using the Microsoft.NET.Sdk.Web SDK...". Maybe it's unnecessary to get that specific. 🤔 Thoughts?

mairaw commented:
@Rick-Anderson @scottaddie can you guys help review this change? 

/cc @KathleenDollard

amaldevv committed:
Updated as per review comments

Made the following changes
1. ovverride -> override
2. are not -> aren't
3. csproj -> .csproj

scottaddie committed:
Adjust formatting

scottaddie committed:
Fix spelling

pkulikov commented on docs/core/tools/dotnet-pack.md:
````diff
@@ -103,6 +103,15 @@ Defines the value for the `$(VersionSuffix)` MSBuild property in the project.
+<PropertyGroup>
+    <IsPackable>true</IsPackable>
+</PropertyGroup>
+```
````
@mairaw will it be displayed nicely? It looks that the xml code block would appear out of the note block.

mairaw replied:
You're right @pkulikov. I think we need to add > until the end of the note include the code sample piece

![image](https://user-images.githubusercontent.com/12971179/44288255-7d545000-a224-11e8-9a6c-6df007d2cddc.png)

mairaw commented on docs/core/tools/dotnet-pack.md:
```diff
@@ -103,6 +103,15 @@ Defines the value for the `$(VersionSuffix)` MSBuild property in the project.
 Sets the verbosity level of the command. Allowed values are `q[uiet]`, `m[inimal]`, `n[ormal]`, `d[etailed]`, and `diag[nostic]`.
 
+> [!NOTE]
+> Web projects aren't packable by default. To override the default behavior, add the below property to your *.csproj* file:
```
below property -> following property

amaldevv committed:
Fixed spelling and formatting

Following changes were made 
below property -> following property
Included code sample inside the note section by adding >

pkulikov commented:
@amaldevv thank you for addressing additional feedback; @mairaw is this PR ready to ship?

mairaw commented:
Sorry I missed this was waiting on me. Merging now. Thanks @amaldevv.
